bin/refresh-common-docker

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# this is a script to refresh the common package
+## before you run this script, make sure you've upped the version number in common/package.json
```

Comment:
I've modified the bash script so that you can pass it a flag indicating whether or not you are in docker. It also checks to make sure that common/package.json has been updated (if it hasn't, this script doesn't really do anything)